pyproject.toml

```diff
@@ -0,0 +1,70 @@
+[tool.poetry]
+name = "scoap3"
+version = "0.1.0"
+description = ""
+authors = ["CERN SIS-TS <cern-sis-ts-admins@cern.ch>"]
+license = "MIT License"
+
+[tool.poetry.dependencies]
+python = "~3.10"
```

Comment:
can't we support python 11? Did you try it?

Reply:
bumped it to python 11 now, docker-compose, the application and the tests still work so I guess we are fine.

Reply:
that's great!